apps/vs-agent/README.md

```diff
@@ -110,9 +109,9 @@ When connecting to other agents, VS-A tries to get information from them in orde
 
 VS-A fetches capabilities from the `discovery.json` file (which is located at at `/www/apps/vs-agent/discovery.json` in the deployed container) to determine available features. If you want to customize the capabilities to look for, replace the volume at this path with your own `discovery.json` file.
 
-### Self VTR
+### Self VR
```

Comment:
I think we must refactor this README to include self tr environment variables as one of the subections of the main VS configuration.

We can also keep the details of the implementation in doc/self-tr-routes and add a link here, but since 99% of the users will be interested only in the config, I don't see why we'll force them to use two documents.